ports/psoc6/mpconfigport.h

```diff
@@ -227,15 +152,17 @@ typedef intptr_t mp_off_t;
 
 #include "shared/runtime/interrupt_char.h"
```

Comment:
Similar to above, couldn't we move the header to the top?

Reply:
done